storage_bucket/delete_file.py

```diff
@@ -46,10 +46,10 @@ def __call__(
         timeout: Optional[TIMEOUT_TYPE] = None,
```

Comment:
Shouldn't it be `**kwargs`?

Reply:
I guess that `TIMEOUT_TYPE ` can be removed from the codebase, now. What do you think?